src/shared/loop-util.c

```diff
@@ -123,6 +123,18 @@ static int loop_configure_verify(int fd, const struct loop_config *c) {
         assert(fd >= 0);
         assert(c);
 
+        if (c->block_size != 0) {
+                int z;
+
+                if (ioctl(fd, BLKSSZGET, &z) < 0)
+                        return -errno;
+
+                assert(z >= 0);
+                if ((uint32_t) z != c->block_size) {
+                        log_debug("LOOP_CONFIGURE doesn't honour .block_size. Continuing.");
+                }
```

Comment:
as per https://systemd.io/CODING_STYLE/#formatting — not {} around single-line if blocks.

Maybe also mention both values here, i.e.

```c
log_debug("LOOP_CONFIGURE didn't honour requested block size %zu, got %i instead. Ignoring.", c->block_size, z);
```

or something like that